packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -722,7 +745,22 @@ function createInstances(cluster: DatabaseClusterNew, props: DatabaseClusterBase
   }
 
   const instanceType = instanceProps.instanceType ?? ec2.InstanceType.of(ec2.InstanceClass.T3, ec2.InstanceSize.MEDIUM);
-  const instanceParameterGroupConfig = instanceProps.parameterGroup?.bindToInstance({});
+
+  if (props.parameterGroup && props.parameters) {
```

Comment:
Should this not be

```suggestion
  if (instanceProps.parameterGroup && instanceProps.parameters) {
```

?